lib/jsdom/living/custom-elements/ElementInternals-impl.js

```diff
@@ -15,6 +19,30 @@ class ElementInternalsImpl {
     return shadow;
   }
 
+  get labels() {
+    if (!this._targetElement._ceDefinition.formAssociated) {
+      throw DOMException.create(this._targetElement._globalObject, [
+        "ElementInternals.labels is only supported for form-associated custom elements",
+        "NotSupportedError"
+      ]);
+    }
+    return NodeList.create(this._targetElement._globalObject, [], {
```

Comment:
We should share the implementation with `HTMLInputElement-impl.js`, by using `getLabelsForLabelable`.